src/playlist.js

```diff
@@ -0,0 +1,66 @@
+var playItem = require('./playitem.js');
+var setupAutoadvance = require('./autoadvance.js');
+
+var playlistMaker = function(player, plist) {
+  var currentIndex = 0;
+  var autoadvanceTimeout = null;
+
+  var playlist = function playlist(list) {
+    if (typeof list === 'number') {
+      return player.playlist.item(list);
```

Comment:
> @heff I'm not adding method to the player itself. Modifying objects you don't own is a bad.

The important thing to me is that overloading the playlist property creates an API that's not consistent with existing APIs, and for something as key as a playlist I'd hope that it would work like other player properties/functions.

What if we prefixed functions with 'playlist', e.g. `player.playlistItem()`?

How is this different from modifying the player object by adding the `playlist` property? Would it help if the playlist plugin included a plugin for each method we were adding to the player object?